In [1]:
import pandas as pd

In [2]:
from event_log_integrity import overlap_metric, events_contained_metric

In [3]:
df_log = pd.read_csv('data/full_event_log_v3.csv', parse_dates=['timestamp', 'timestamp_end'])

In [5]:
overlap_metric(df_log);
events_contained_metric(df_log);

Overlap Metric: 0.61%
(258923, 9)
Entirely contained events: 1.86%


Esto dice que en un ~0.61% de las actividades en este caso (`case_id` como `truck_id`) la actividad siguiente comienza, mientras no termina la anterior. De la misma manera, las actividades contenidas en otras, es decir que comienzan después pero que terminan antes que la primera es de un 1.86%.